db-checkbook/README.md

```diff
@@ -0,0 +1,11 @@
+# Checkbook NYC Exploratory Work and Join to CPDB
+
+## To do:
+- upload relevant Checkbook NYC data to Digital Ocean
+- write the code to pull down data from Digital Ocean
+- Dea and Ali make their own branches and start working on smallest units of extractable code from our original notebooks, checking it into our projects, implementing, then submitting PR requests
+- mimic utility of methods from db-facilities/facdb/cli.py (init, dataloading, build, etc) using facdb terminology
```

Comment:
Was chatting with @fvankrieken, and our thought is that it might be easiest to (for now) just mimic the terminology of facdb, but putting the actual data manipulation steps in separate, sequential files, so that you two can avoid any merge conflicts. So you'd end up with a few python scripts like 
- 01_dataloading.py
- 02_build.py
etc. Though it's up to you!

Reply:
that sounds good to me - where in the current folder structure would it be most appropriate to have those scripts? python folder, or?

Reply:
python folder sounds good to me. Sticks with convention